lib/Mojolicious/Plugin/Model.pm

```diff
@@ -10,6 +10,10 @@ our $VERSION = '0.11';
 sub register {
   my ($plugin, $app, $conf) = @_;
 
+  # check if need camelize moniker
+  my $path = $app->home . '/lib/' . $app->moniker;
+  $app->moniker(camelize($app->moniker)) unless -d $path;
```

Comment:
Why do we need to change `moniker` globally?

Reply:
Changing globally might not be the best solution, I'm going to change the code.